lib/pg_audit_log/extensions/postgresql_adapter.rb

```diff
@@ -30,29 +30,39 @@ def rename_table(table_name, new_name)
     end
   end
 
-  def reconnect!
+  def reconnect!(...)
```

Comment:
Starting with Rails 7.1, this takes kwargs: https://github.com/rails/rails/blob/v7.1.5.1/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb#L680

I used the Ruby [argument forwarding](https://blog.saeloun.com/2019/12/04/ruby-2-7-adds-new-operator-for-arguments-forwarding/) (`...`) syntax throughout these monkey-patched methods, since we just want all args to be passed along via `super` to the method being monkey-patched.